core/src/main/java/org/apache/druid/math/expr/Parser.java

```diff
@@ -60,13 +65,32 @@
       }
     }
     FUNCTIONS = ImmutableMap.copyOf(functionMap);
+
+    Map<String, ApplyFunction> applyFunctionMap = new HashMap<>();
+    for (Class clazz : ApplyFunction.class.getClasses()) {
+      if (!Modifier.isAbstract(clazz.getModifiers()) && ApplyFunction.class.isAssignableFrom(clazz)) {
+        try {
+          ApplyFunction function = (ApplyFunction) clazz.newInstance();
+          applyFunctionMap.put(StringUtils.toLowerCase(function.name()), function);
+        }
+        catch (Exception e) {
+          log.info("failed to instantiate " + clazz.getName() + ".. ignoring", e);
```

Comment:
I know its a copy/paste from function, but I think it should be termed as error.  Also, "e" should be first argument , currently I think it is just getting ignored.

Reply:
still cleaning stuff up, will be sure to get this :+1: